### PR TITLE
Fix filenames with incorrect symbols for Windows

### PR DESCRIPTION
**Pull request by dmikhalin**
## What is the goal of this PR?

We had some filenames in generated AsciiDoc files that are prohibited in Windows. It affected only NodeJS Driver, but cloning the repo was impossible in Windows. We fixed it.

## What are the changes implemented in this PR?

We replaced symbols `"<"` and `">"` with underscores.

We also fixed the script for docs testing: now we add all new generated files to git.

### CODE REVIEW

File: nodejs/docs/Stream<T>.adoc
Comment (krishnangovindraj) [inline nodejs/docs/Stream<T>.adoc:1]: Since this file is being deleted, shouldn't a file named `Stream_T_.adoc` be added?

Comment (dmikhalin) [reply nodejs/docs/Stream<T>.adoc:1]: Definitely. And it should be tested, but somehow it isn't...

**Review by krishnangovindraj — APPROVED** [review]
Looks good!

**Comment by flyingsilverfin** [comment]
We should regenerate the docs after this change no?

**Comment by krishnangovindraj** [comment]
@flyingsilverfin I think he did regenerate the docs. If he hadn't, the test would fail.

**Comment by dmikhalin** [comment]
> We should regenerate the docs after this change no?

@flyingsilverfin We have done it. But only one file is affected.

**Pull request merged**